_data/projects/mediawiki.yml

```diff
@@ -0,0 +1,12 @@
+name: MediaWiki
+desc: The collaborative editing software that runs Wikipedia
+site: https://www.mediawiki.org
+tags:
+- mediawiki
+- wiki
+- web
+- cms
+- php
+upforgrabs:
+  name: Easy
+  link: https://phabricator.wikimedia.org/tag/easy/
```

Comment:
Would it be better to navigate the user straight to the task list? e.g. https://phabricator.wikimedia.org/maniphest/query/PaVmrjYb6zFl/#R